lib/con_cache/operations.ex

```diff
@@ -20,10 +20,21 @@ defmodule ConCache.Operations do
   defp lock_pid(cache, key),
     do: elem(cache.lock_pids, :erlang.phash2(key, tuple_size(cache.lock_pids)))
 
-  def get(cache, key) do
+  def get(cache, key, opts \\ []) do
     case fetch(cache, key) do
```

Comment:
Get is implemented on top of fetch, so I think you'd need to do this in `fetch`. Because otherwise, `ConCache.fetch_or_store` won't work.